gnosis/safe-react
# feat: Dashboard pending txs

## What it solves
Resolves #3699

## How this PR fixes it
Creates a component that displays one of the two scenarios:
1. Fetches all local Safes and displays aggregated `AWAITING_CONFIRMATION` transactions.
2. Displays all the aggregated `AWAITING_CONFIRMATION` transactions awaiting confirmations for all the Safes owned by the connected account.

Both scenarios take you to the respective Safe "Queue List" on clicking a listed Safe.


## How to test it
Navigate to `.../app/home` and observe how "Transactions to Sign" widget is populated.
Click one of the list items to proceed to the "Queue List"

## Screenshots
<img width="335" alt="Screen Shot 2022-03-23 at 15 51 29" src="https://user-images.githubusercontent.com/32431609/159730492-9705d506-2aa5-4be4-9d4b-fd5eb2192b98.png">

<img width="330" alt="Screen Shot 2022-03-23 at 15 51 41" src="https://user-images.githubusercontent.com/32431609/159730553-d340e470-86bb-4f00-a57d-83997459ea5f.png">
